src/core/i18n/globalMessageIds.ts

```diff
@@ -6,6 +6,12 @@ export default makeMessages('glob', {
     edit: m('Editing'),
     readonly: m('Read-only'),
   },
+  genderOptions: {
+    f: m('Male'),
+    m: m('Female'),
```

Comment:
Isn't `f` supposed to be Female and `m` supposed to be Male?

Reply:
no it's the Zetkin Mandatory Genderfuck Policy in operation 😛

Reply:
Haha, thanks for noticing this. 💯